release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -237,6 +237,10 @@ In {product-title} 4.12, you can do the following from the *Helm* page:
 * View the list of the existing Helm chart repositories with their scope in the *Repositories* page.
 * View the newly created Helm release in the *Helm Releases* page.
 
+[id="ocp-4-12-negativematcher-alert"]
+===== Negative matchers in Alertmanager
+With this update, Alertmanager now supports `Negative matcher` option. Using `Negative matcher`, you can update the *Label value* to a not equals matcher. The negative matcher checkbox changes `=` (value equals) into `!=` (value does not equal) and changes `=~` (value matches regex) into `!~`(value does not match Regex). Also, the "Use RegEx" checkbox label is renamed to "RegEx".
```

Comment:
```suggestion
With this update, Alertmanager now supports a `Negative matcher` option. Using `Negative matcher`, you can update the *Label value* to a Not Equals matcher. The negative matcher checkbox changes `=` (value equals) into `!=` (value does not equal) and changes `=~` (value matches regular expression) into `!~`(value does not match regular expression). Also, the *Use RegEx* checkbox label is renamed to *RegEx*.
```
- add an indefinite article to first sentence
- Per the SSG: Replace `regex` with `regular expression` (https://redhat-documentation.github.io/supplementary-style-guide/#_r)
- Per IBM Style: UI elements, including text labels, need to be in bold text (https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#references-to-interface-elements)